pac4j-http/src/main/java/org/pac4j/http/credentials/authenticator/DigestAuthenticator.java

```diff
@@ -0,0 +1,15 @@
+package org.pac4j.http.credentials.authenticator;
+
+import org.pac4j.http.credentials.DigestCredentials;
+import org.pac4j.http.credentials.UsernamePasswordCredentials;
+
+/**
+ * This interface represents the contract to validate digest credentials.
+ *
+ * @author Mircea Carasel
+ */
+public interface DigestAuthenticator extends Authenticator<DigestCredentials> {
```

Comment:
Also... my understanding is that the validate call has the purpose to create HttpProfile and set it in Credentials...